shared-data/js/constants.ts

```diff
@@ -732,7 +735,7 @@ export const COMBO_FIXTURES: CutoutFixtureIdsWithFakes[] = [
   STAGING_AREA_RIGHT_SLOT_FIXTURE,
 ]
 
-export const DEFAULT_AA_FOR_WASTE_CHUTE = '96ChannelWasteChute'
+export const DEFAULT_AA_FOR_WASTE_CHUTE = '1ChannelWasteChute'
```

Comment:
Sorry, just for my own edification, why is this change beneficial exactly?

Reply:
waste chute addressable areas has multiple options per fixture.
at first we choose the 96 bc it appears in the waste chute w/o the cover. we now decided that we do support waste chute with covers so I had to change the default to a aa that exists in all compatible fixture options. 
makes sense?

Reply:
This was a bit confusing to me as well. Can we do something like this instead so we know this is a reference to an AA constant? 

```suggestion
export const DEFAULT_AA_FOR_WASTE_CHUTE = ONE_CHANNEL_WASTE_CHUTE_ADDRESSABLE_AREA
```